app/src/main/res/layout-hdpi/activity_multimeter.xml

```diff
@@ -256,13 +252,11 @@
 
             <View
                 android:id="@+id/knob_marker"
-                android:layout_width="@dimen/multimeter_knob_width"
-                android:layout_height="@dimen/multimeter_knob_height"
+                android:layout_width="160dp"
```

Comment:
This is what is known as **hardcoding**. And we don't recommend that.

Reply:
i will resolve this